helpers.js

```diff
@@ -382,7 +382,23 @@ function doReport(config, objects, errors) {
         const uniqueIssues = eslintHelpers.getUniqueIssues(eslintIssuesByBaseName);
 
         const formatter = getFormatter(config.style);
-        config.logger.log(formatter(uniqueIssues));
+        let formatterResult;
+        if (config.style === 'html') {
```

Comment:
Do we need to check the style here? Can we use the same `rulesMeta` for whatever style is chosen?

Reply:
Sure. It won't break anything on other formatters. I've updated PR. So please look at it